### PR TITLE
first cut of wallet list storybook

### PR DESCRIPTION
**Pull request by zanderz**
Not sure why you have to click "Add a Wallet" to see the menu even though the storybook sets `showingMenu={true}`
Also, I copied Nojima in using the handwave icon until we have a wallet one.

screen shot 2018-04-30 at 4 06 12 pm

### CODE REVIEW

File: shared/stellar/walletList/index.stories.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    name: 'Second wallet',
+    contents: '56.9618203 XLM',
+  },
+]
+
+const load = () => {
+  storiesOf('Stellar', module).add('Wallet List', () => (
+    <Box style={{width: 240}}>
+      {mocks.map(m => <Wallet key={m.name} {...m} />)}
+      <AddWallet showingMenu={true} onAddNew={() => {}} onLinkExisting={() => {}} />
```
Comment (chrisnojima) [inline shared/stellar/walletList/index.stories.js]: We should use action() and not empty callbacks

File: shared/stellar/walletList/index.stories.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import {storiesOf} from '../../stories/storybook'
+import {Wallet, AddWallet} from '../../wallets/wallet'
+
+const common = {
+  isSelected: false,
+  name: '',
+  keybaseUser: '',
+  contents: '',
+  onSelect: () => {},
+}
```
Comment (chrisnojima) [inline shared/stellar/walletList/index.stories.js]: Action()

File: shared/wallets/wallet/index.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    const amountStyle = {
+      color: this.props.isSelected ? globalColors.white : globalColors.black_40,
+      backgroundColor: color,
+      fontSize: 11,
+    }
+    const props = this.props
+    return (
+      <ClickableBox onClick={props.onSelect} style={{backgroundColor: color}}>
+        <WalletBox style={{...rowContainerStyle, backgroundColor: color}}>
+          <Box style={iconBoxStyle}>
```
Comment (chrisnojima) [inline shared/wallets/wallet/index.js]: Let's use box2

File: shared/wallets/wallet/index.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+          position="bottom center"
+        />
+      </ClickableBox>
+    )
+  }
+}
+
+const AddWallet = FloatingMenuParentHOC(_AddWallet)
+
+const walletRowStyle = {
```
Comment (chrisnojima) [inline shared/wallets/wallet/index.js]: We need to start to use stylesheeecreate

**Comment by zanderz** [comment]
Thanks for the pointers. I switched everything from Box to Box2 and nothing lines up right, so this is still WIP for now.

**Comment by zanderz** [comment]
ok now it looks pretty reasonable I think

File: shared/wallets/wallet/index.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    const props = this.props
+    return (
+      <ClickableBox onClick={props.onSelect} style={{backgroundColor: color}}>
+        <Box2 style={{height: rowHeight, backgroundColor: color}} direction="horizontal" fullWidth={true}>
+          <Box2 direction="horizontal">
+            <Icon
+              type={'iconfont-hand-wave'}
+              color={globalColors.darkBlue}
+              fontSize={28}
+              style={{marginLeft: globalMargins.tiny, marginRight: globalMargins.tiny}}
```
Comment (chrisnojima) [inline shared/wallets/wallet/index.js]: i think this margin is too large

File: shared/wallets/wallet/index.js [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+            />
+          </Box2>
+          <Box2 direction="vertical">
+            <Box2 direction="horizontal" fullWidth={true}>
+              {this.props.keybaseUser && (
+                <Avatar
+                  size={16}
+                  username={this.props.keybaseUser}
+                  style={{marginRight: globalMargins.xtiny}}
```
Comment (chrisnojima) [inline shared/wallets/wallet/index.js]: box2 has a gap property whose purpose is to remove all these margins inside of components to space them

File: shared/wallets/wallet/index.js [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+  render() {
+    return (
+      <ClickableBox
+        onClick={this.props.toggleShowingMenu}
+        style={{backgroundColor: globalColors.white}}
+        ref={this.props.setAttachmentRef}
+      >
+        <Box2 style={{height: rowHeight}} direction="horizontal" fullWidth={true}>
+          <Box2 direction="vertical">
```
Comment (chrisnojima) [inline shared/wallets/wallet/index.js]: shouldn't need these inner box2s

**Pull request merged**